packages/ui-dom-utils/src/findFocusable.ts

```diff
@@ -83,12 +83,7 @@ function findFocusable(
 
 function hidden(element: Element | Node) {
   const cs = getComputedStyle(element)
-  return (
```

Comment:
This refers to a [ticket](https://instructure.slack.com/archives/C0JCJ63TR/p1682957023840189) on the Slack channel. After researching the issue, I discovered that the developers who were implementing the feature had left. Therefore, I decided to remove the feature and test how it affected the components. Overall, everything was fine in the manual test. I suggest writing unit tests.